desktop_version/src/editor.h

```diff
@@ -113,6 +113,8 @@ class editorclass{
   std::vector<std::string> directoryList;
   std::vector<LevelMetaData> ListOfMetaData;
 
+  const char *linebreak;
```

Comment:
This attribute should be initialized in `editorclass::reset()`, where all the other attributes of `editorclass` are initialized.